interfaces/cython/cantera/thermo.pyx

```diff
@@ -66,22 +66,23 @@ cdef class Species:
             section='species')
 
     """
-    def __cinit__(self, *args, init=True, **kwargs):
+    def __cinit__(self, name=None, composition=None, *args, init=True, **kwargs):
         if init:
-            self._species.reset(new CxxSpecies())
+            if name is not None:
+                name = stringify(name)
+            else:
+                name = stringify("")
+            if composition is not None:
+                composition = comp_map(composition)
+            else:
+                composition = comp_map("")
+            self._species.reset(new CxxSpecies(name, composition))
             self.species = self._species.get()
 
-    def __init__(self, name=None, composition=None, charge=None, size=None,
-                 *args, init=True, **kwargs):
+    def __init__(self, name=None, composition=None, charge=None, size=None, *args, init=True, **kwargs):
```

Comment:
I don't undesrtand why this is being changed.

Reply:
A mistake! 😄 I had moved `name` and `composition` out of this initializer, which allowed everything to fit on one line, but then I added them back and didn't fix the line length 🤦‍♂️